RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -13,25 +13,16 @@ void print_footer() {
 
 boolean auto_pre_adventure()
 {
-	auto_log_debug("Running auto_pre_adv.ash");
```

Comment:
This made some sense before where we had that heavy rains abort.
But now the immediate next action is to report that we are starting or skipping the preadventure script (depending on whether auto_disableAdventureHandling is set or not)